package.json

```diff
@@ -1118,56 +1118,94 @@
     "walkthroughs": [
       {
         "id": "confluent-walkthrough",
-        "title": "Connect to Kafka Clusters",
+        "title": "Get Started with Confluent for VS Code",
         "description": "Interact with Apache Kafka® clusters running locally or on Confluent Cloud.",
         "steps": [
           {
-            "id": "connect-local-cluster",
-            "title": "Connect to Kafka",
-            "description": "**Confluent Cloud Clusters** \nLog in to your Confluent account to see your Confluent Cloud resources. \n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)\n**Local Clusters** \nIf you have Kafka running locally, it appears automatically in the Resources view. \n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "id": "connect-conflulent-cloud",
+            "title": "Connect to Confluent Cloud",
+            "description": "Log in to see your Confluent Cloud resources.\n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
             "media": {
               "image": "resources/walkthrough/connect.png",
               "altText": "Connect to Kafka clusters"
             },
             "completionEvents": [
-              "onCommand:command:confluent.connections.ccloud.signIn",
-              "onCommand:confluent.docker.startLocalResources"
+              "onCommand:command:confluent.connections.ccloud.signIn"
             ]
           },
           {
             "id": "create-connections",
-            "title": "Connect directly to Kafka / Schema Registry",
-            "description": "To connect to an Apache Kafka or Schema Registry cluster, click $(plus) next to Resource > Other. The cluster can be in Confluent Cloud, Confluent Platform, or hosted elsewhere.\n[Open Connection Form](command:confluent.connections.direct)",
+            "title": "Connect Directly",
+            "description": "To connect to a Confluent Platform, WarpStream, Apache Kafka, and/or Schema Registry clusters.\n Select the $(plus) icon next to Resource to create or import a connection.\n[Create Connection](command:confluent.connections.direct)",
```

Comment:
We could add "Confluent Cloud (via API Credentials)" or just "Confluent Cloud" here.

Reply:
Added "Confluent Cloud (via API keys)".

Reply:
Fixed.